pages/requests/index.js

```diff
@@ -8,17 +8,26 @@ import {
   RequestList,
   Title,
 } from '@scientist-softserv/webstore-component-library'
-import { configureErrors, dark, useAllRequests } from '../../utils'
+import {
+  configureErrors,
+  dark,
+  useDefaultWare,
+  useAllRequests 
+} from '../../utils'
 
 // TODO(alishaevn): authenticate the user
 const Requests = ({ ...props }) => {
   const router = useRouter()
   const { data: session } = useSession()
-  const { requests, isLoading, isError } = useAllRequests()
+  const { requests, isLoadingAllRequests, isAllRequestsError } = useAllRequests()
   const { user, userError, userLoading } = props
+  const { defaultWareID, isLoadingDefaultWare, isDefaultWareError } = useDefaultWare('make-a-request')
```

Comment:
the slug will always be `make-a-request` right? so we could hard code that in the `useDefaultWare` function.

Reply:
will it for sure always be make-a-request? or do suppliers ever call it something different?

Reply:
ill just hard code it for now and i guess its easy enough to change if the slug varies at all by supplier

Reply:
I believe that the default ware is actually scientist controlled. not something that each supplier sets up. so I'd imagine that it's always the same. 
but agreed that we can change it if not.